pkg/agent/route/route_windows.go

```diff
@@ -36,6 +37,17 @@ const (
 	outboundFirewallRuleName = "Antrea: accept packets to local Pods"
 )
 
+var (
+	// ServiceGWVirtualIP is a virtual IP for redirecting ClusterIP Service traffic from host network to OVS.
+	// The traffic will be routed to antrea-gw0 via the virtual IP. It is added to use a neighbor cache entry
```

Comment:
It is added to use a single neighbor cache entry for the traffic (rather than one entry for each ClusterIP).

Reply:
Updated.